README.md

```diff
@@ -8,10 +8,10 @@ $ npx create-react-app my-new-app
 $ cd my-new-app
 ```
 
-From here to help installing Shift.js you can use the install_script.sh script found in the repository.
+From the app directory you can install Shift.js with:
 
 ```shell
-$ ../path/to/shiftjs/install_script.sh
+$ npx @binaris/shift-react-app
```

Comment:
This should probably be a part of the shift-cli: `npx @binaris/shift-cli shiftify-react-app`.

Reply:
Well it seems like a flaw in the spec to me.